benchmarks/sharded-bm/README.md

```diff
@@ -127,12 +127,10 @@ gcloud compute instances list --project=nearone-mocknet --filter <UNIQUE ID>
 <!-- cspell:words BENCHNET -->
 ```sh
 export CASE=cases/forknet/10_cp_1_rpc_10_shard/
-export VIRTUAL_ENV=<absolute path to virtual env bin directory>
-export SYNTH_BM_BIN=<absolute path or URL to near-synth-bm binary>
```

Comment:
I guess we can remove line 124
`2. Make sure you have a `near-synth-bm` binary that can be run on a mainnet node.`